src/modules/75-cf/components/FlagArchiving/useArchiveFlagDialog.tsx

```diff
@@ -46,36 +107,65 @@ const useArchiveFlagDialog = ({
     }
   }
 
+  const getDialogContent = () => {
+    if (state === STATUS.error)
+      return <PageError message={showError(dependentFlagsError)} onClick={() => refetchDependentFlags()} />
```

Comment:
Try to avoid multiline things like this without brackets. A quick visual scan would make you think that the next statements were inside this `if` block.
```suggestion
    if (state === STATUS.error) {
      return <PageError message={showError(dependentFlagsError)} onClick={() => refetchDependentFlags()} />
    }
```